tests/core/pyspec/eth2spec/debug/random_value.py

```diff
@@ -51,65 +51,68 @@ def get_random_ssz_object(rng: Random,
     """
     if chaos:
         mode = rng.choice(list(RandomizationMode))
-    if issubclass(typ, ByteList):
-        # ByteList array
-        if mode == RandomizationMode.mode_nil_count:
-            return typ(b'')
-        elif mode == RandomizationMode.mode_max_count:
-            return typ(get_random_bytes_list(rng, min(max_bytes_length, typ.length)))
-        elif mode == RandomizationMode.mode_one_count:
-            return typ(get_random_bytes_list(rng, min(1, typ.length)))
-        elif mode == RandomizationMode.mode_zero:
-            return typ(b'\x00' * min(1, typ.length))
-        elif mode == RandomizationMode.mode_max:
-            return typ(b'\xff' * min(1, typ.length))
-        else:
-            return typ(get_random_bytes_list(rng, rng.randint(0, min(max_bytes_length, typ.length))))
-    elif issubclass(typ, ByteVector):
+    # if issubclass(typ, ByteList):
```

Comment:
Why removal here?

Reply:
See above, ByteList is out for now, `List[byte, N]` suffices.

Reply:
Probably best to make sure this type still shows up in test generators whenever dev gets released next

Reply:
ByteList is back with a new implementation. But that said, it's not used in Phase0, so I am not too worried.